internal/connectors/plugins/public/powens/client/bank_accounts.go

```diff
@@ -25,39 +12,5 @@ type BankAccount struct {
 	ConnectionID int      `json:"id_connection"`
 	Currency     Currency `json:"currency"`
 	OriginalName string   `json:"original_name"`
-}
-
-func (c *client) GetBankAccount(ctx context.Context, accessToken string, bankAccountID int) (BankAccount, error) {
```

Comment:
that change is a bit more extreme than what I thought 😂 We rely 100% on webhooks I guess?